packages/contracts-bedrock/src/universal/FeeVault.sol

```diff
@@ -17,20 +18,32 @@ abstract contract FeeVault {
     }
 
     /// @notice Minimum balance before a withdrawal can be triggered.
+    ///         Use the `minWithdrawalAmount()` getter as this is deprecated
+    ///         and is subject to be removed in the future.
+    /// @custom:legacy
     uint256 public immutable MIN_WITHDRAWAL_AMOUNT;
 
-    /// @notice Wallet that will receive the fees.
+    /// @notice Account that will receive the fees. Can be located on L1 or L2.
+    ///         Use the `recipient()` getter as this is deprecated
+    ///         and is subject to be removed in the future.
+    /// @custom:legacy
     address public immutable RECIPIENT;
 
-    /// @notice Network which the RECIPIENT will receive fees on.
+    /// @notice Network which the recipient will receive fees on.
+    ///         Use the `withdrawalNetwork()` getter as this is deprecated
+    ///         and is subject to be removed in the future.
+    /// @custom:legacy
     WithdrawalNetwork public immutable WITHDRAWAL_NETWORK;
 
     /// @notice The minimum gas limit for the FeeVault withdrawal transaction.
-    uint32 internal constant WITHDRAWAL_MIN_GAS = 35_000;
+    uint32 internal constant WITHDRAWAL_MIN_GAS = 400_000;
 
     /// @notice Total amount of wei processed by the contract.
     uint256 public totalProcessed;
 
+    /// @notice Reserve extra slots in the storage layout for future upgrades.
+    uint256[49] private __gap;
```

Comment:
Should the gap size be 48, this way a contract inheriting from this starts at slot 50, as opposed to starting at 51? Starting at 50 feels more analogous to starting at zero. I'm not sure what we prefer to do and IIRC we may have been inconsistent in at least one contract.

Reply:
I am down for making this 48, not super opinionated. It may not be consistent everywhere in the codebase